### PR TITLE
Add carbon-charts w/ npm auto-update

### PR DESCRIPTION
**Pull request by nstuyvesant**

### CODE REVIEW

**Review by nstuyvesant — COMMENTED** [review]
Sorry I left out the extension.

File: packages/c/carbon-charts.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "filename": "dist/umd/bundle.umd.js",
+  "autoupdate": {
+    "source": "npm",
+    "target": "@carbon/charts",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.css",
+          "**/*.css.map"
```
Comment (MattIPv4) [inline packages/c/carbon-charts.json:47]:
```suggestion
          "*.css?(.map)"
```
You can combine this into a single glob, and I don't believe you need a glob star here, as the CSS files aren't in nested folders?

File: packages/c/carbon-charts.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        ]
+      },
+      {
+        "basePath": "dist/umd",
+        "files": [
+          "**/*.@(js|map)"
```
Comment (MattIPv4) [inline packages/c/carbon-charts.json:53]:
```suggestion
          "*.js?(.map)"
```
Same deal here?

File: packages/c/carbon-charts.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		"data",
+		"typescript"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://www.carbondesignsystem.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core"
+  },
+  "filename": "dist/umd/bundle.umd.js",
```
Comment (MattIPv4) [inline packages/c/carbon-charts.json:38]:
```suggestion
  "filename": "bundle.umd.js",
```
As you're setting `basePath`, we'll pull files up out of that directory

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**